packages/paste-website/src/components/tokens-example/index.tsx

```diff
@@ -17,10 +17,10 @@ export const ColorBox: React.FC<BackgroundColor> = ({backgroundColor}) => {
   return <Absolute backgroundColor={backgroundColor} padding="space50" preset="fill" />;
 };
 
-type BorderBoxProps = Pick<BoxProps, 'borderColor' | 'borderWidth'>;
-export const BorderBox: React.FC<BorderBoxProps> = ({borderColor, borderWidth}) => {
+type BorderBoxProps = Pick<BoxProps, 'borderColor' | 'borderWidth' | 'height'>;
```

Comment:
I think we can simplify this further actually:
1- Remove this `| 'height'`
2- Remove `, height` on line 21
3- Remove `height={height}` on line 23
4- Add a hardcoded `padding="space60"` on line 23 (like how ShadowBox does on line 93, making the style consistent).

This has the added benefit of fixing border-widths too:
<img width="684" alt="image" src="https://user-images.githubusercontent.com/1592327/67599027-a62f7980-f734-11e9-90c8-22925332863b.png">

Reply:
Sounds good!